### ISSUE TITLE
Should we have voice over option when extensions get installed/downloaded?

### ISSUE THREAD

**Issue by kapilraghuwanshi** [microsoft/vscode#97126]
We can have **voice over** checkbox option when extensions get installed in our VS code. So, the user can hear that aloud that -

> "Specific extension A gets installed successfully". 

It should be a checkbox option on _Extension settings (... on top right)_ where user can opt in or not, by default could be untick.

**Comment by isidorn** [comment]
@kapilraghuwanshi thanks for the suggestion. However this should not be an option, but by default we should announce when an extension gets installed.
I tried this out and currently we do not announce anything.
@sandy081 is it possible to do a simple `alert` when an extension gets installed. Something like
`Extensions NAME installed`
Same when extension gets uninstalled.

**Comment by sandy081** [comment]
@isidorn We already announce that [here](https://github.com/microsoft/vscode/blob/71a587e2cb8af5b7bea4cf699f1f4425acf5e207/src/vs/workbench/contrib/extensions/browser/extensionsActions.ts#L236-L236) using an alert. Let me know if there is something missing here?

**Comment by isidorn** [comment]
@sandy081 thanks for the pointer. The first time I tried it I could not hear the announcment, now it seems to be ok.
@kapilraghuwanshi can you please provide steps which you do when you do not hear the Announcment that the extension got installed?

**Comment by kapilraghuwanshi** [comment]
@isidorn  Can you please share the steps for that? Because I am not hearing anything after installing any extensions. Do we have any additional settings to be turned on for that?

**Comment by isidorn** [comment]
@kapilraghuwanshi if you are not using Linux it should just work out of the box, without any settings.
So Ctrl + Shift + X to open extensions vielwet, type something, install via button (by tabbing onto it or by clicking). It should read that the extension got installed
@sandy081 can you also please try, since this sometimes is not read on macOS for me
@kapilraghuwanshi what os do you use?

**Comment by sandy081** [comment]
![image](https://user-images.githubusercontent.com/10746682/82192560-71fbf000-98f4-11ea-885d-9b5fbd0f7e88.png)

@isidorn I tried with few extensions and I see above message is read. When are you not seeing this?

**Comment by kapilraghuwanshi** [comment]
@sandy081 I use macOS only.

My suggestion to add in VS Code is that - VS Code should say it loudly that "Extension A has been installed successfully" like we have these features in PyCharm.

**Comment by isidorn** [comment]
@kapilraghuwanshi well as @sandy081 points out we nicely read this out.
Can you also reproduce your problem using vscode insiders https://code.visualstudio.com/insiders/
